status/worksets.py

```diff
@@ -38,7 +38,7 @@ def get(self, worksets='all'):
         t = self.application.loader.load("worksets.html")
         ws_data=self.worksets_data()
         headers= [['Date Run', 'date_run'],['Workset Name', 'workset_name'], \
-                 ['Projects (samples)','projects'], ['Operator', 'technician'],\
+                 ['Projects (samples)','projects'], ['Date finished', 'finish date'],['Operator', 'technician'],\
```

Comment:
should it be `finish_date` with a `_` ? (not a stopper, just asking as every other dual word key have a `_`) :smile_cat: 

Maybe I can merge and you can change it later if needed ? :eyes: